### ISSUE TITLE
Pinocchio `getNumpyType` is not found in `def zero`

### ISSUE THREAD

**Issue by cmastalli** [stack-of-tasks/pinocchio#1090]
Hi guys,

I have installed the latest devel branch of Pinocchio.
And when I run the following script
```python
import pinocchio
vec = pinocchio.utils.zero(3)
```
I got the error message:
```bash
/usr/local/lib/python2.7/dist-packages/pinocchio/utils.py in zero(n)
     38 
     39 def zero(n):
---> 40     if pin.getNumpyType()==np.matrix:
     41         return np.matrix(np.zeros([n, 1] if isinstance(n, int) else n))
     42     else:
```

Could it be that I need to update EigenPy and Pinocchio is not detecting this?

Yours,
Carlos

**Comment by cmastalli** [comment]
**Update**
When I compiled the master branch of EigenPy everything works well.

The cmake needs to be updated in order to request for the right version of EigenPy. Unfortunately, I don't know which one is the minimal one.

**Comment by jcarpent** [comment]
What is your old version of EigenPy?

**Comment by cmastalli** [comment]
> What is your old version of EigenPy?

I think that was v1.6.1, however I am not sure about it.

**Comment by jcarpent** [comment]
Solved by #1091.
